charts/ingress-nginx/tests/controller-poddisruptionbudget_test.yaml

```diff
@@ -71,3 +71,30 @@ tests:
     asserts:
       - hasDocuments:
           count: 0
+
+  - it: should create a PodDisruptionBudget with `IfHealthyBudget` unhealthyPodEvictionPolicy
+    set:
+        controller.replicaCount: 2
+        controller.unhealthyPodEvictionPolicy: IfHealthyBudget
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: PodDisruptionBudget
+      - equal:
+          path: spec.unhealthyPodEvictionPolicy
+          value: IfHealthyBudget
+
+  - it: should create a PodDisruptionBudget with `AlwaysAllow` unhealthyPodEvictionPolicy
+    set:
+      controller.replicaCount: 2
+      controller.unhealthyPodEvictionPolicy: AlwaysAllow
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: PodDisruptionBudget
+      - equal:
+          path: spec.unhealthyPodEvictionPolicy
+          value: AlwaysAllow
+
```

Comment:
This one is obsolete as it's just a different string value and we don't have any logic that's changing what you put into the value or changes the output depending on the value.